lib/liquid/combinators/tags/case.ex

```diff
@@ -26,7 +26,15 @@ defmodule Liquid.Combinators.Tags.Case do
   @type t :: [
           case: [
             variable: LexicalToken.value(),
-            clauses: [String.t() | [when: [conditions: [body: Liquid.t()]]]]
+            clauses: [
+              String.t()
+              | [
+                  when: [
+                    conditions: [LexicalToken.value() | {:logical, [or: LexicalToken.value()]}],
```

Comment:
try changing this:
`conditions: [LexicalToken.value() | {:logical, [or: LexicalToken.value()]}],`
to this:
`conditions: [LexicalToken.value(), optional({:logical, [or: LexicalToken.value()]})]`